### PR TITLE
Issue-170/ Refactor Footer for decent mobile view

### LINKED ISSUE #170: Refactor Footer.jsx so it includes an appropriate view for small screens
`Footer.jsx` component was only developed with tablet+ sized screens in mind.  Need to add in a more appropriate view for larger screens.

### PR DESCRIPTION
**Pull request by xscottxbrownx**
**Description**
---
This PR creates a decent rendering of the current version of the `footer` for small screens.

**Updates**
---
Screenshot 2023-05-21 at 12 49 15 PM
Screenshot 2023-05-21 at 12 49 28 PM
Screenshot 2023-05-21 at 12 49 44 PM

**Related Issue**
---
Closes Issue #170 

**Future Thoughts**
---
- Still need a finalized design and thoughts on what should be in the footer, but this is a decent skeleton for now

### CODE REVIEW

**Comment by leekahung** [comment]
Not bad, the code looks alright as well.

But I'm curious how it'll look like with what we have with the two section footer, so from horizontal to vertical when it hits a breakpoint for mobile similar to that mid section in the three section footer.

File: src/components/Footer/Footer.jsx
Comment (leekahung) [inline src/components/Footer/Footer.jsx]: Think we could also optimize these Links like what you've done in `<RenderCompanyInfoSection />` with map

Comment (xscottxbrownx) [reply src/components/Footer/Footer.jsx]: Complete

**Comment by xscottxbrownx** [comment]
> Not bad, the code looks alright as well.
> 
> But I'm curious how it'll look like with what we have with the two section footer, so from horizontal to vertical when it hits a breakpoint for mobile similar to that mid section in the three section footer.

Pull latest changes and check it.  Same for mobile, but back to the horizontal layout for larger screens
Screenshot 2023-05-21 at 4 38 18 PM
Screenshot 2023-05-21 at 4 39 21 PM

**Comment by leekahung** [comment]
Like PR #131, I think looks ready for merging. Going to hold this up at the moment until #157 is completed, but will approve afterwards. Great work!

**Review by leekahung — APPROVED** [review]
I'm okay with this as mentioned from before

File: src/components/Footer/Footer.jsx
Comment (leekahung) [inline src/components/Footer/Footer.jsx]: Oh, didn't catch this the last time I've reviewed it, but we need a key attribute for the different elements. Just a React thing. Think we can use `key={link.title}`.

File: src/components/Footer/Footer.jsx
Comment (leekahung) [inline src/components/Footer/Footer.jsx]: Just checked this attribute for Divider on MUI. It looks like it suppose to just be a boolean for true/false. I think you can change this to `flexItem={isReallySmallScreen ? null : true}`

**Review by leekahung — CHANGES_REQUESTED** [review]
Sorry for the late catch, just found it while I was resolving the merge conflicts. But it's only two things and think it'll help clear up the error messages that popup related to React/MUI. See feedback.

File: src/components/Footer/Footer.jsx
Comment (xscottxbrownx) [reply src/components/Footer/Footer.jsx]: Not sure how I missed that! Good eyes.

Completed.

File: src/components/Footer/Footer.jsx
Comment (xscottxbrownx) [reply src/components/Footer/Footer.jsx]: Switched to your suggestion.

**Review by leekahung — APPROVED** [review]
Alright, I'm good with these changes. Approved for merging.

**Pull request merged**
